.jshintrc

```diff
@@ -1,22 +1,21 @@
 {
-  "bitwise":    true,   // prevent bitwise operations (e.g. ^, |, &)
-  "camelcase":  true,   // all vars must use camelCase
-  "browser":    true,   // allow browser global vars
-  "curly":      true,   // require curly brackets in loops and conditionals
-  "eqeqeq":     true,   // require strict equality
-  "forin":      true,   // require hasOwnProperty check in for...in loops
-  "freeze":     true,   // prevent native object prototype overwriting
-  "immed":      true,   // require immediate function invocations to be wrapped in parens
-  "indent":     2,      // require 2-space indentation
-  "newcap":     true,   // require PascalCase constructor function names
-  "noarg":      true,   // prevent arguments.callee, because it sucks
-  "undef":      true,   // prevent explicitly undeclared variables
-  "unused":     true,   // warn when defining variables that are never used
-  "strict":     true,   // use strict mode
-  "node":       true,   // allow node global vars
-  "latedef":    true,   // don't use variables before they're defined
+  "bitwise":    true,
+  "camelcase":  true,
+  "browser":    true,
+  "curly":      true,
+  "eqeqeq":     true,
+  "forin":      true,
+  "freeze":     true,
+  "immed":      true,
+  "indent":     2,
+  "newcap":     true,
+  "noarg":      true,
+  "undef":      true,
+  "unused":     true,
+  "strict":     true,
+  "node":       true,
+  "latedef":    true,
```

Comment:
Because comments are not valid json, and a jsonlinter gets angry at it.  Don't you have a jsonlinter in sublime?  :p

Also, all the comments were doing is repeating what the setting was.  Writing 2, and then two again in longer English, isn't really what comments are for.  If they were saying _why_ it's 2, then I'd be much more interested.